From: Fabian Vogt <fabian@ritter-vogt.de>
Subject: Add back HostKeyAlgorithms=+ssh-dsa

The previous patch removed the old option, presumably by accident.
Also fix the bugref.

Index: kdeconnect-kde-21.08.2/plugins/sftp/mounter.cpp
===================================================================
--- kdeconnect-kde-21.08.2.orig/plugins/sftp/mounter.cpp
+++ kdeconnect-kde-21.08.2/plugins/sftp/mounter.cpp
@@ -129,7 +129,8 @@ void Mounter::onPackageReceived(const Ne
         << QStringLiteral("-o") << QStringLiteral("IdentityFile=") + KdeConnectConfig::instance().privateKeyPath()
         << QStringLiteral("-o") << QStringLiteral("StrictHostKeyChecking=no") //Do not ask for confirmation because it is not a known host
         << QStringLiteral("-o") << QStringLiteral("UserKnownHostsFile=/dev/null") //Prevent storing as a known host
-        << QStringLiteral("-o") << QStringLiteral("HostKeyAlgorithms=+ssh-rsa") //https://bugs.kde.org/show_bug.cgi?id=351725
+        << QStringLiteral("-o") << QStringLiteral("HostKeyAlgorithms=+ssh-dss") //https://bugs.kde.org/show_bug.cgi?id=351725
+        << QStringLiteral("-o") << QStringLiteral("HostKeyAlgorithms=+ssh-rsa") //https://bugs.kde.org/show_bug.cgi?id=443155
         << QStringLiteral("-o") << QStringLiteral("PubkeyAcceptedKeyTypes=+ssh-rsa")
         << QStringLiteral("-o") << QStringLiteral("uid=") + QString::number(getuid())
         << QStringLiteral("-o") << QStringLiteral("gid=") + QString::number(getgid())
